pages/community/[community].tsx

```diff
@@ -21,16 +21,12 @@ export const getServerSideProps = compose<
     list: CommunityMember[];
     community: string;
   }
->(
-  cache(),
-  errorLogger,
-  translator(i18n),
-  async ({ params: { community } = {} }) => {
-    const list = await new CommunityMemberModel().getAll({ community });
-
-    return { props: JSON.parse(JSON.stringify({ list, community })) };
-  },
-);
+>(errorLogger, translator(i18n), async ({ params: { community } = {} }) => {
+  const list = (await new CommunityMemberModel().getAll({ community })).filter(
+    member => !!member.approver,
+  );
```

Comment:
```suggestion
  const list = await new CommunityMemberModel().getAll({ community });
```
在下方加配置即可，从后端接口查询的源头来过滤掉：

https://github.com/kaiyuanshe/kaiyuanshe.github.io/blob/20ce808f9a6c1b38e1954bf02dc0dde55a1ed786/models/Community/CommunityMember.ts#L38-L38